src/pages/Workspace.tsx

```diff
@@ -95,6 +97,7 @@ const CanvasLayer: React.FC = () => {
             wheel={{ step: 0.15 }}
             velocityAnimation={{ equalToMove: false }}
         >
+            {!project?.config.elements?.length && <p className="top-12 left-28 fixed font-bold text-2xl z-[1]">Right click to add Instruments</p>}
```

Comment:
Think it can be better, but we'll roll with it for now.

Reply:
Oh for sure, but I just wanted to put out *something* cuz I was quite confused at first.